.travis.yml

```diff
@@ -37,6 +37,7 @@ before_script:
 
 script:
 - mvn clean package
+- mvn javadoc:javadoc
```

Comment:
This additional command should work fine, but I think if you had changed the `package` goal in the first command to be `install` that it would also create the javadocs (I think... it might also require an additional pom.xml config change as well, not 100% sure)).   Just for future reference :)

Reply:
Oh, probably not then 😕  I wasn't sure originally if that was the issue we were having.